access_indexor.sol

```diff
@@ -1,4 +1,4 @@
-pragma solidity 0.4.24;
+pragma solidity >0.4.24;
```

Comment:
Shouldn't we pin this to the new version (5.4 - right?)
Solidity hasn't been stable between releases so pinning it seems to be the safest way to avoid problems

Reply:
Done